Styling/README.md

```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
 }
 ```
 
+#### atan2
+
+`atan2(value1 : Number, value2 : Number) : Number`
+
+Returns the angle `theta` in polar coordinates `(r, theta)` for a point with Cartesian coordinates `(value1, value2)`.
```

Comment:
Also like the other math functions make sure to mention the result is in radians.